tox.ini

```diff
@@ -3,7 +3,7 @@
 minversion = 3.7.0
 isolated_build = true
 envlist=
-  py{27,35,36,37,38}-pytest{310,46,54,latest}
+  py{36,37,38,39,310}-pytest{310,46,54,latest}
```

Comment:
Pytest versions in that line need to add the 6.x one

Reply:
Good catch, uppdated! Should there also be `main` for `pytestmain`?